docs/FAQ.md

```diff
@@ -21,33 +21,85 @@
 
 You would use KUDO when `kubectl apply -f` isn't quite enough to manage your application. If you are just applying the same YAML on updates you probably won't need the extra business logic KUDO gives you.
+Kudo should be used any time you would use an Operator.  It can provide an advanced user experience, automating such features as updates, backups and scaling.
 
 #### What is a Framework
 
-`Framework` is the highlevel CRD object. An example for a `Framework` is e.g. the [kafka-framework.yaml](https://github.com/kudobuilder/frameworks/blob/master/repo/stable/kafka/versions/0/kafka-framework.yaml) that you find in the [kudobuilder/frameworks](https://github.com/kudobuilder/frameworks) repo.
+`Framework` is the high-level description of a deployable service which is represented as an CRD object. An example `Framework` is the [kafka-framework.yaml](https://github.com/kudobuilder/frameworks/blob/master/repo/stable/kafka/versions/0/kafka-framework.yaml) that you find in the [kudobuilder/frameworks](https://github.com/kudobuilder/frameworks) repository.
+
+Kafka Framework Example
+```
+apiVersion: kudo.k8s.io/v1alpha1
+kind: Framework
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: kafka
+```
+
+More examples can be found in the https://github.com/kudobuilder/frameworks project and include: [flink](https://flink.apache.org/), [kafka](https://kafka.apache.org/), and [zookeeper](https://zookeeper.apache.org/).
+
 
 #### What is a FrameworkVersion?
 
-A `FrameworkVersion` is the particular implementation of a `Framework`. It contains: 
+A `FrameworkVersion` is the particular implementation of a `Framework` containing:
 
 - [Parameters](#what-is-a-parameter)
 - [Plans](#what-is-a-plan)
 - Kubernetes Objects
 
-An example for a `FrameworkVersion` is e.g. the [kafka-frameworkversion.yaml](https://github.com/kudobuilder/frameworks/blob/master/repo/stable/kafka/versions/0/kafka-frameworkversion.yaml) that you find in the [kudobuilder/frameworks](https://github.com/kudobuilder/frameworks) repo.
+An example for a `FrameworkVersion` is the [kafka-frameworkversion.yaml](https://github.com/kudobuilder/frameworks/blob/master/repo/stable/kafka/versions/0/kafka-frameworkversion.yaml) that you find in the [kudobuilder/frameworks](https://github.com/kudobuilder/frameworks) repository.
+
+```
+apiVersion: kudo.k8s.io/v1alpha1
+kind: FrameworkVersion
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: kafka-2.11-2.4.0
+  namespace: default
+spec:
+  serviceSpec:
+  version: "2.11-2.4.0"
+  connectionString: ""
+  framework:
+    name: kafka
+    kind: Framework
+  parameters:
+    - name: BROKER_COUNT
+      description: "Number of brokers spun up for Kafka"
+      default: "3"
+      displayName: "Broker Count"
+      ...
+plans:
+   deploy:
+     strategy: serial
+     phases:
+       - name: deploy-kafka
+         strategy: serial
+         steps:
+           - name: deploy
+             tasks:
+               - deploy      
+```
```

Comment:
should this live here or shall we link to the actual file to avoid running out of sync?

Reply:
we do have a link ^^ however... I thought it is easier to have a sample front and center.  It helps to conceptualize the concept.  It doesn't have to be exact IMO.   It is easier to conceptualize without jumping all around.    (Not a strong opinion tho)